src/python/WMCore/ReqMgr/Service/RequestAdditionalInfo.py

```diff
@@ -208,4 +248,45 @@ def post(self, name):
             # it doesn't work
             url = "%s/%s" % (self.reqdb_url, name)
             result = helper.saveCouchUrl(url)    
-        return result
\ No newline at end of file
+        return result
+    
+    
+class SplittingAlgos(RESTEntity):
+    
+    def __init__(self, app, api, config, mount):
+        # main CouchDB database where requests/workloads are stored
+        RESTEntity.__init__(self, app, api, config, mount)
+        self.reqdb_url = "%s/%s" % (config.couch_host, config.couch_reqmgr_db)
+    
+    
+    def _validate_args(self, param, safe):
+        # TODO: need proper validation but for now pass everything
+        args_length = len(param.args)
+        if args_length > 0 :
```

Comment:
Isn't it `if args_length < 1:`?
BTW, for my education, besides handling different validations (which is not the case for this class), what's the motivation not to put the actual validation code into the validate() method?